state-chain/pallets/cf-ingress-egress/src/tests/boost.rs

```diff
@@ -942,13 +936,70 @@ fn stop_boosting_is_disabled_by_safe_mode() {
 		assert_ok!(IngressEgress::stop_boosting(
 			RuntimeOrigin::signed(BOOSTER_1),
 			eth::Asset::Eth,
-			BoostPoolTier::FiveBps
+			TIER_5_BPS
 		));
 		assert_eq!(
-			BoostPools::<Test, ()>::get(eth::Asset::Eth, BoostPoolTier::FiveBps)
+			BoostPools::<Test, ()>::get(eth::Asset::Eth, TIER_5_BPS)
 				.unwrap()
 				.get_available_amount_for_account(&BOOSTER_1),
 			None
 		);
 	});
 }
+
+#[test]
+fn test_create_boost_pools() {
+	new_test_ext().execute_with(|| {
+		// Make sure the pools do not exists already
+		assert!(BoostPools::<Test, ()>::get(eth::Asset::Eth, TIER_5_BPS).is_none());
+		assert!(BoostPools::<Test, ()>::get(eth::Asset::Eth, TIER_10_BPS).is_none());
+		assert!(BoostPools::<Test, ()>::get(eth::Asset::Flip, TIER_5_BPS).is_none());
+
+		// Create all 3 pools in one go
+		assert_ok!(Pallet::<Test, _>::create_boost_pools(
+			RuntimeOrigin::signed(ALICE),
+			vec![
+				BoostPoolId { asset: eth::Asset::Eth, tier: TIER_5_BPS },
+				BoostPoolId { asset: eth::Asset::Eth, tier: TIER_10_BPS },
+				BoostPoolId { asset: eth::Asset::Flip, tier: TIER_5_BPS },
+			]
+		));
+
+		// Check they now exist
+		assert!(BoostPools::<Test, ()>::get(eth::Asset::Eth, TIER_5_BPS).is_some());
+		assert!(BoostPools::<Test, ()>::get(eth::Asset::Eth, TIER_10_BPS).is_some());
+		assert!(BoostPools::<Test, ()>::get(eth::Asset::Flip, TIER_5_BPS).is_some());
+
+		// Check that all 3 emitted the creation event
+		assert_event_sequence!(
+			Test,
+			RuntimeEvent::IngressEgress(Event::BoostPoolCreated {
+				boost_pool: BoostPoolId { asset: eth::Asset::Eth, tier: TIER_5_BPS },
+			}),
+			RuntimeEvent::IngressEgress(Event::BoostPoolCreated {
+				boost_pool: BoostPoolId { asset: eth::Asset::Eth, tier: TIER_10_BPS },
+			}),
+			RuntimeEvent::IngressEgress(Event::BoostPoolCreated {
+				boost_pool: BoostPoolId { asset: eth::Asset::Flip, tier: TIER_5_BPS },
+			})
+		);
+
+		// Should not be able to create the same pool again
```

Comment:
Let's also check that we didn't accidentally removed the existing pool (e.g. that we didn't set the pool to None in `try_mutate_exists`).